aten/src/ATen/native/SharedReduceOps.h

```diff
@@ -114,19 +114,19 @@ struct WelfordOps {
   }
 };
 
-template <typename acc_t, typename factor_t>
+template <typename acc_t, typename data_t, typename out_t, typename factor_t>
 struct MeanOps {
   factor_t factor;
 
-  inline C10_DEVICE acc_t reduce(acc_t a, acc_t b, int64_t /*idx*/) const {
+  inline C10_DEVICE acc_t reduce(acc_t a, data_t b, int64_t /*idx*/) const {
     return combine(a, b);
```

Comment:
`combine` takes 2 `acc_t` types as input, but you are passing `acc_t` and `data_t` now. will it auto cast `data_t` to `acc_t`? can it cause any issues?

Reply:
Thanks your comments. Fixed it.